mscorlib/system/exception.cs

```diff
@@ -36,14 +36,21 @@ namespace System {
     [ComDefaultInterface(typeof(_Exception))]
     [Serializable]
     [ComVisible(true)]
-    public class Exception : ISerializable, _Exception
-    {
+#if MONO
+    [StructLayout (LayoutKind.Sequential)]
+#endif
+    public class Exception : ISerializable
+#if !(MONO && MOBILE)
+        , _Exception
```

Comment:
This looks a little awkward but I did it this way to force a merge conflict if Microsoft makes changes to the Exception definition in future.